source/interface/transport_interface.h

```diff
@@ -163,7 +163,11 @@ typedef struct NetworkContext NetworkContext_t;
  * @param[in] pBuffer Buffer to receive the data into.
  * @param[in] bytesToRecv Number of bytes requested from the network.
  *
- * @return The number of bytes received or a negative error code.
+ * @return The number of bytes received or a negative value to indicate
```

Comment:
Is this change mentioned in the commit message?

Reply:
Nope, but this change was just to satisfy the SSoT CI check between `transport_interface.h` files of coreMQTT and coreHTTP